ansible/roles/ci/doc/setup/circle.md

```diff
@@ -0,0 +1,19 @@
+# Circle CI Server
+
+## How to setup
+
+The following repository can be used as an example of [Circle CI](https://circleci.com/) configuration [build servers check](https://github.com/shadow-robot/build-servers-check)
+
+The process of setting up [Circle CI](https://circleci.com/) server
+
+  * In the root folder of your repository
+    * Copy *circle.yml* from [build servers check](https://github.com/shadow-robot/build-servers-check) repository. 
+    * Create *repository.rosinstall* in case if you want to install some packages from source code. This file should be in in [rosinstall](http://wiki.ros.org/rosinstall) format.
```

Comment:
Create *repository.rosinstall* in case you want to install some packages from source code. This file should be in [rosinstall](http://wiki.ros.org/rosinstall) format.